node/resolvers/catalog/facets.ts

```diff
@@ -27,6 +27,7 @@ const formatCategoriesTree = (root: any) => {
 
 const addSelected = (
   facets: any[],
+  // eslint-disable-next-line no-shadow
```

Comment:
In this case, is better to solve the problem than simply disabling the rule.

A function called map is already being imported, but the parameter has the same name, so there is this conflict, to resolve this, you can do this:

```ts
type AddSelected = (
  facets: any[],
  options: { query: string; map: string }
) => any

const addSelected: AddSelected = (facets, options) => {
  return facets.map((facet: any) => {
    let children = facet.Children

    if (children) {
      children = addSelected(children, options)
    }

    const currentFacetSlug = decodeURIComponent(facet.Value).toLowerCase()

    const isSelected =
      zip(
        options.query.toLowerCase().split('/').map(decodeURIComponent),
        options.map.toLowerCase().split(',')
      ).find(
        ([slug, slugMap]) => slug === currentFacetSlug && facet.Map === slugMap
      ) !== undefined

    return {
      ...facet,
      Children: children,
      selected: isSelected,
    }
  })
}
```